src/Oliver.Client/Executing/LogSender.cs

```diff
@@ -26,7 +26,7 @@ public LogSender(IRestClient restClient, ILogger<LogSender> logger)
             Task.Run(QueueListener, this.cancellation.Token);
         }
 
-        public async Task LogStep(long executionId, int stepId, bool isLastStep = false, List<string> logs = null, CancellationToken cancellationToken = default)
+        public Task LogStep(long executionId, int stepId, bool isLastStep = false, List<string> logs = null, CancellationToken cancellationToken = default)
```

Comment:
It would be great if you add `Async` suffix to all methods which return `Task` type even if they aren't async at all as the current method.

Reply:
done